src/modules/75-cf/pages/onboarding/__test__/OnboardingDetailPage.test.tsx

```diff
@@ -43,8 +48,20 @@ describe('OnboardingDetailPage', () => {
   })
 
   test('OnboardingDetailPage empty state should be rendered properly', () => {
+    ;(usePreferenceStore as jest.Mock).mockImplementation(() => {
+      return {
+        setPreference: jest.fn(),
+        preference: true,
+        clearPreference: jest.fn
```

Comment:
Better not to mock in each test if it's always the same, but just a wee thing... when using `jest.fn`, you need to call it, not just reference it.
```suggestion
        clearPreference: jest.fn()
```

Reply:
Thanks